src/components/Editor/EditorToolbar.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback } from "react";
+import { useToolbarActions, usePlaygroundArgs } from "@/hooks";
+import EditorToolbarButton from "./EditorToolbarButton";
+import EditorToolbarDivider from "./EditorToolbarDivider";
+import {
+  useAddonState,
+  useParameter,
+  useStorybookApi,
+} from "@storybook/manager-api";
+import {
+  ADDON_ID_FOR_PARAMETERS,
+  DEFAULT_ADDON_STATE,
+  PANEL_ID,
+  DEFAULT_ADDON_PARAMETERS,
+} from "@/consts";
+import { PlaygroundState } from "@/types";
+
+const EditorToolbar: React.FC = () => {
+  const { selectStory } = useStorybookApi();
+  const { playgroundStoryId } = useParameter(
+    ADDON_ID_FOR_PARAMETERS,
+    DEFAULT_ADDON_PARAMETERS
+  );
+
+  const { code, updateCode, resetCode } = usePlaygroundArgs();
+  const [state, setState] = useAddonState<PlaygroundState>(
+    PANEL_ID,
+    DEFAULT_ADDON_STATE
+  );
+  const onFontSizeChange = useCallback(
+    (amount: number) => {
+      setState((state) => ({
+        ...state,
+        fontSize: Math.max(12, Math.min(18, state.fontSize + amount)),
+      }));
+    },
+    [setState]
+  );
+  const { onCopy, isCopied, shouldAllowCopy, onFormatCode, onReset } =
+    useToolbarActions(code, updateCode, resetCode, state.selectedTab);
+
+  return (
+    <div
+      className="toolbar"
+      style={{
+        position: "relative",
+        display: "flex",
+        alignItems: "center",
+        gap: 4,
+        padding: 4,
+        borderBottom: "1px solid rgba(255, 255, 255, 0.1)",
+      }}
+    >
+      <EditorToolbarButton
+        tooltip="Show playground view"
+        icon="beaker"
+        onClick={() => selectStory(playgroundStoryId)}
+      />
+      <EditorToolbarButton
+        tooltip={shouldAllowCopy ? "" : "Editor is empty"}
+        text={isCopied ? "Copied!" : "Copy"}
+        icon={isCopied ? "check" : "copy"}
+        color={isCopied ? "green" : undefined}
```

Comment:
```suggestion
        color={isCopied ? "green" : null}
```

Reply:
once you pass undefined is like you didn't pass any prop, and this is what I intended to do here